### ISSUE TITLE
Choice Field Group - docs indicate to nest div directly beneath ul. Is there supposed to be li in between?

### ISSUE THREAD

**Issue by jbarr33** [OfficeDev/office-ui-fabric-js#282]
I tried it but there doesn't seem to be a style to remove the rendered bullets from the `li`

Thanks.

**Comment by Jahnp** [comment]
Hi @jbarr33, just to be clear, are you referring to the documentation on [this page for the ChoiceFieldGroup component](https://dev.office.com/fabric-js/Components/ChoiceFieldGroup/ChoiceFieldGroup.html)? Looking there, I see this markup:

```html
<div class="ms-ChoiceFieldGroup" id="choicefieldgroup" role="radiogroup">
  <div class="ms-ChoiceFieldGroup-title">
    <label class="ms-Label is-required">Unselected</label>
  </div>
  <ul class="ms-ChoiceFieldGroup-list">
    <div class="ms-RadioButton">
     ...
``` 

Which is indeed not valid markup; semantically only `li` elements should be direct children of `ul` elements. I'm not sure if this was a bug/oversight when creating the sample code or if the CSS for ChoiceFieldGroup was originally structured that way, but either way this should be investigated & fixed. The solution could either be turning the `ms-RadioButton` element into an `li` (less likely) or instead wrapping in an `li` with a class like ms-ChoiceFieldGroup-listItem`. 

We'll try to look into this soon, but feel free to do the same--if you happen to resolve this, feel free to open a PR fixing the documentation. Sorry for the hassle!

**Comment by leddie24** [comment]
Fixed in #284
